src/rules/table-column-count.js

```diff
@@ -30,10 +33,28 @@ export default {
 		messages: {
 			inconsistentColumnCount:
```

Comment:
```suggestion
			extraCells:
```

I think we can rename it to `extraCells` here. Previously, `inconsistentColumnCount` made sense, but now we have two messages with conflicting meanings.